modules/RTC/RTC.js

```diff
@@ -102,7 +102,8 @@ function _newCreateLocalTracks(mediaStreamMetaData = []) {
             sourceType,
             stream,
             track,
-            videoType: videoType || null
+            videoType,
```

Comment:
Why did you remove the `null`? Aren't we checking for `null` somewhere?

Reply:
By mistake, corrected